### PR TITLE
[EM-4916]-MV E2E Tests Code Optimisation

### PR DESCRIPTION
**Pull request by phani-perla1**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EM-4916

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "e2e": "protractor --webdriverUpdate=false",
     "e2e:local": "yarn webdriver:update && protractor e2e/local.protractor.conf.js",
     "e2e:fullfunctional_PT": "yarn webdriver:update && protractor e2e/ci.protractor.conf.js",
-    "e2e:fullfunctional": "NODE_PATH=. node ./node_modules/codeceptjs/bin/codecept.js run -c ./test/end-to-end/ --grep @ci --steps --reporter mocha-multi",
+    "e2e:fullfunctional": "NODE_PATH=. node ./node_modules/codeceptjs/bin/codecept.js run -c ./test/end-to-end/ --grep @Test99 --steps --reporter mocha-multi",
```
Comment (mdayican) [inline package.json:46]: is it just for one test?

File: test/end-to-end/helpers/mvCaseHelper.js [hunk 2]
```diff
@@ -95,51 +94,60 @@ async function pdfAndImageRotationTest(I, caseId, mediaType) {
...
   }
+
+  // if (process.env.TEST_URL.split('-')[3] === 'pr' || await getEnvironment() === 'local') {
+  //   await navigatePreviewUrl(I);
+  // } else {
+  //   await openCaseDocumentsInMediaViewer(I, caseId, mediaType);
+  // }
+  // await I.clearBookMarks();
+  // await I.createBookMark();
+  // await I.updateBookMarks();
```
Comment (mdayican) [inline test/end-to-end/helpers/mvCaseHelper.js:140]: you can remove instead of comment outs.

File: test/end-to-end/helpers/mvCaseHelper.js [hunk 3]
```diff
@@ -95,51 +94,60 @@ async function pdfAndImageRotationTest(I, caseId, mediaType) {
...
   }
+
+
+  // if (process.env.TEST_URL.split('-')[3] === 'pr' || await getEnvironment() === 'local') {
+  //   await navigatePreviewUrl(I);
+  // } else {
+  //   await openCaseDocumentsInMediaViewer(I, caseId, mediaType);
+  // }
+  // await I.clearBookMarks();
+  // await I.createBookMark();
```
Comment (mdayican) [inline test/end-to-end/helpers/mvCaseHelper.js:113]: you can remove instead of comment outs.

Comment (phani-perla1) [reply test/end-to-end/helpers/mvCaseHelper.js:113]: Sure I will do

**Pull request closed without merging**